src/App.vue

```diff
@@ -161,75 +178,92 @@
         this.$root.tabTheme = to
       }
     },
+    updated() {
+      if (this.$isWebApp) this.filterLinksAndImages(window.document.body)
+    },
     mounted () {
       this.$nextTick(() => this.updateAppIcon())
+      const user_path_target = window.location.pathname
 
       this.$localData.root.TABS_SWITCH_TO()
+      // Switch to the last tab (good) but replaces history (so we use the previously captured value)
 
-      electron.webFrame.setZoomFactor(1)
+      if (user_path_target != this.$localData.root.activeTabObject.url) {
```

Comment:
URL navigation: on the web, the user can manually poke at the address bar to insert whatever location they want; this consumes that info and folds it into tab logic